### DISCUSSION TITLE
nx for dart

### DISCUSSION THREAD

**Discussion by sultanmyrza** [nrwl/nx#5028] [Feature Requests]
Nx is great tool ❤️.

I use it for my side project
React (for admin panel)
Ionic + Capasitor (for users) 
Nestjs (for api) and wrap it to cloud functions 😅

What if we had nx for dart?
then I would do 
Flutter (web app for admin panel)
Flutter (mobile app for users) 
Aquaduct or something (for api) and wrap it to cloud functions 😅

**Comment by theoludwig** [comment]
Hey! @sultanmyrza 
There is actually a plugin to do flutter development inside a `@nrwl/nx` monorepo. (I didn't test it myself)
See: https://github.com/tinesoft/nxrocks/tree/master/packages/nx-flutter

**Comment by tinesoft** [comment]
Hi @sultanmyrza!

I'm Tine, the author of that `@nxrocks/nx-flutter` plugin @Divlo  mentioned 😇 . 
It does indeed support creating Flutter `application`, `package`, `module` or `plugin`.
It is being used successfully by a [couple of projects](https://github.com/tinesoft/nxrocks/network/dependents) already 🥰 

 Please, feel free to try it and let me know any issue/improvement you may have.

Happy Hacking!
